docs/how-to-guides/themes/theme-json.md

```diff
@@ -991,6 +992,21 @@ Currently block variations exist for "header" and "footer" values of the area te
 }
 ```
 
+### patterns
+
+<div class="callout callout-alert">
+This field is only allowed when the Gutenberg plugin is active. It will be added in WordPress 6.0 so in previous versions it will be ignored.
```

Comment:
> This field is only allowed when the Gutenberg plugin is active. It will be added in WordPress 6.0 so in previous versions it will be ignored.

What do you think of this rewording:

```suggestion
This field requires the Gutenberg plugin active and using the [version 2](https://developer.wordpress.org/block-editor/reference-guides/theme-json-reference/theme-json-living/) of `theme.json`.
```

Reply:
Thank you!